components/director/cmd/ns-adapter/README.md

```diff
@@ -0,0 +1,313 @@
+# NS-Adapter
+
+## Overview
+
+NS-Adapter listens for incoming **reports** on on-premice systems from external Notification Service. Based on the data from the report systems may be created, updated or deleted.
+
+## Details
+
+This section describes the API schema that a Notification Service must implement to integrate with the NS-Adapter.
+
+### Authorization
+
+Valid JWT token with "tenant.consumerTenant" and "tenant.externalTenant" claims. ConsumerTenant and externalTenant should contain the internal ID and the external ID from existing business_tenant_mappings record in Compass database.
+
+### Reports
+
+There are two types of reports:
+- Delta:
+  - systems that are __present__ in the report but are __unknown__ to Compass will be created
+  - systems that are __present__ in the report and are __known__ to Compass will be updated
+  - systems that are __not present__ in the report but are __known__ to Compass will be deleted
+- Full:
+  - same as Delta
+  - systems from Compass labeled with `SCC` are deleted if the value of their label is not present in the report - there is no entity in the `value` array from the request body which has `subaccount` and `locationId` matching these from the systems `SCC` label
```

Comment:
What's the difference between this line and the one before?
`systems that are __not present__ in the report but are __known__ to Compass will be deleted`